.circleci/config.yml

```diff
@@ -0,0 +1,63 @@
+# Java Maven CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-java/ for more details
+
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/openjdk:8-jdk
+    working_directory: ~/repo
+    environment:
+      MAVEN_OPTS: -Xmx3200m
+    steps:
+      - checkout
+      - run: git submodule sync
```

Comment:
@pselle Should we do a throwdown about git submodules? I see the use case here, but they add complexity.